webrtc.html

```diff
@@ -1327,9 +1327,10 @@
             <code>none</code>. This call will result in a change to the
             connection state of the ICE Agent, and may result in a change to
             media state if it results in different connectivity being
-            established. The only members of the candidate attribute
+            established. The only members of <code>candidate</code>
             used by this method are <var>candidate</var>, <var>sdpMid</var> and
-            <var>sdpMLineIndex</var>; the rest are ignored.</p>
+            <var>sdpMLineIndex</var>; the rest are ignored. A value for either
+            <var>sdpMid</var> or <var>sdpMLineIndex</var> MUST be provided.</p>
```

Comment:
It would be great to have this requirement as a part of the algorithm steps below and throw a TypeError if not fulfilled. The resulting error type can be discussed, but we have earlier used TypeError when we put custom requirements on dictionaries to align with the new "required member" feature.